assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceTiles.js

```diff
@@ -0,0 +1,349 @@
+/**
+ * AudienceTiles component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import {
+	DATE_RANGE_OFFSET,
+	MODULES_ANALYTICS_4,
+} from '../../../datastore/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import AudienceTile from './AudienceTile';
+import {
+	BREAKPOINT_SMALL,
+	BREAKPOINT_TABLET,
+	useBreakpoint,
+} from '../../../../../hooks/useBreakpoint';
+import { Tab, TabBar } from 'googlesitekit-components';
+import InfoTooltip from '../../../../../components/InfoTooltip';
+import AudienceTooltipMessage from './AudienceTooltipMessage';
+
+const { useSelect } = Data;
+
+export default function AudienceTiles( { Widget } ) {
+	const [ activeTile, setActiveTile ] = useState( 0 );
+	const breakpoint = useBreakpoint();
+	const isTabbedBreakpoint =
+		breakpoint === BREAKPOINT_SMALL || breakpoint === BREAKPOINT_TABLET;
+
+	// An array of audience resource names.
+	const configuredAudiences = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getConfiguredAudiences()
+	);
+	const audiencesDimensionFilter = {
+		filter: {
+			fieldName: 'audienceResourceName',
+			inListFilter: {
+				values: configuredAudiences,
+			},
+		},
```

Comment:
I've just noticed there's a mismatch in the syntax here - this is directly using the underlying report syntax rather than that of our abbreviated report options. It should be as follow; I'll apply the change here, as well as to `getReportForAllAudiences()` to save a round trip.

```suggestion
		audienceResourceName: configuredAudiences,
```